unlock-app/src/services/web3Service.js

```diff
@@ -183,6 +183,7 @@ export default class Web3Service {
           callback(transaction)
         } else if (event === 'NewLock') {
           return this.getLock(args.newLockAddress).then((lock) => {
+            lock.lockName = newLock.lockName
```

Comment:
Maybe add a comment here to indicate that since the lock's name is not stored on chain we need to pass if from the arugments provided...